src/scenes/AcceleratorRouter/Cohorts/CohortForm.tsx

```diff
@@ -102,29 +104,47 @@ export default function CohortForm<T extends CreateCohortRequestPayload | Update
             borderRadius: theme.spacing(4),
             padding: theme.spacing(3),
           }}
-          width={isMobile ? '100%' : '700px'}
+          width={'100%'}
         >
-          <Grid item xs={12} sx={{ marginTop: theme.spacing(2) }}>
-            <Textfield
-              errorText={validateFields && !localRecord?.name ? strings.REQUIRED_FIELD : ''}
-              id='name'
-              label={strings.NAME}
-              onChange={(value) => updateField('name', value)}
-              required
-              type='text'
-              value={localRecord.name}
-            />
+          <Grid container spacing={theme.spacing(3)} width={'100%'}>
+            <Grid item xs={isMobile ? 12 : 4} sx={{ marginTop: theme.spacing(2) }}>
+              <Textfield
+                errorText={validateFields && !localRecord?.name ? strings.REQUIRED_FIELD : ''}
+                id='name'
+                label={strings.NAME}
+                onChange={(value) => updateField('name', value)}
+                required
+                type='text'
+                value={localRecord.name}
+              />
+            </Grid>
+            <Grid item xs={isMobile ? 12 : 4} sx={{ marginTop: theme.spacing(2) }}>
+              <Dropdown
+                fullWidth={true}
+                label={strings.CURRENT_PHASE}
+                onChange={(value) => updateField('phase', value)}
+                options={currentPhaseDropdownOptions}
+                required
+                selectedValue={localRecord.phase}
+              />
+            </Grid>
           </Grid>
-          <Grid item xs={12} sx={{ marginTop: theme.spacing(2) }}>
-            <Dropdown
-              fullWidth={true}
-              label={strings.CURRENT_PHASE}
-              onChange={(value) => updateField('phase', value)}
-              options={currentPhaseDropdownOptions}
-              required
-              selectedValue={localRecord.phase}
+
+          {/* TODO: uncomment this section once createdTime & modifiedTime are available in Cohort records */}
```

Comment:
Create a ticket to follow up (here and the other TODOs in this PR)?

Reply:
You got it 👍